Contributing-code-to-Oppia.md

```diff
@@ -18,6 +19,13 @@ If you run into any problems along the way, we're here to help! Check out our [[
 * [Tips for Success](#tips-for-success)
 * [Notes](#notes)
 
+## How to find information on this wiki
```

Comment:
Since the title is "How to find information on this wiki" do you want to include some info about the sidebar also?

Reply:
Which sidebar? Sidebar that shown after search on github or sidebar that our wiki have?

Reply:
Done.